### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by PavlovaOlena**
[DEMO LINK](https://PavlovaOlena.github.io/layout_moyo-header/)
[TEST REPORT LINK](https://PavlovaOlena.github.io/layout_moyo-header/report/html_report/)

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -1 +1,53 @@
-/* Styles go here */
+body,
+html {
+  padding: 0;
+  margin: 0;
+  border: 0;
```
Comment (qoa11a) [inline src/style.css:5]: Why do you reset padding and border? Do body and HTML have them?

**Review by qoa11a — CHANGES_REQUESTED** [review]
Add a box-shadow to header according to the design.

File: src/index.html [hunk 2]
```diff
@@ -7,8 +7,61 @@
...
+    <header>
+      <div class="container">
+        <label class="link__home nav__item">
+          <a href="#home">
+            <img src="images/logo.png" alt="logo">
+          </a>
+        </label> 
+        <nav class="navigation"> 
+          <ul class="nav__list">
+            <label>
```
Comment (witflash) [inline src/index.html:22]: You don't need an additional `label` wrapper here.
Follow the next structure:
```
<ul>
  <li>
    <a>
  </li>
  ...
  <li>
    <a>
  </li>
</ul>
```

File: src/index.html [hunk 3]
```diff
@@ -7,8 +7,61 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link href="https://fonts.googleapis.com/css2?family=IBM+Plex+Serif:wght@400;700&family=Montserrat:wght@400;700&family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <div class="container">
+        <label class="link__home nav__item">
```
Comment (witflash) [inline src/index.html:15]: Don't use `label` here. Use `div` instead if you need any wrapper.

File: src/index.html [hunk 4]
```diff
@@ -7,8 +7,61 @@
...
+          <a href="#home">
+            <img src="images/logo.png" alt="logo">
+          </a>
+        </label> 
+        <nav class="navigation">
```
Comment (witflash) [inline src/index.html:20]: It should be `<nav class="nav">` if you use `nav__list` naming for BEM elements.

File: src/index.html [hunk 5]
```diff
@@ -7,8 +7,61 @@
...
+        <label class="link__home nav__item">
+          <a href="#home">
+            <img src="images/logo.png" alt="logo">
+          </a>
+        </label> 
+        <nav class="navigation"> 
+          <ul class="nav__list">
+            <label>
+              <li class="nav__item is-active">
+                <a class="link__nav" href="#apple">Apple</a>
```
Comment (witflash) [inline src/index.html:24]: According to the BEM methodology, this element class name should be `nav__link`.

**Review by witflash — CHANGES_REQUESTED** [review]
Let's try to fix all tests.

1. Add padding for the `<a>` elements instead of `<li>`.
image

2. Use `position: absolute` for the `after` pseudo element and `relative` for the `<a>`.

3. Remove the left padding for the first item and the right padding for the last item:
image

If you still have any questions feel free to ask in `fe_chat` in Slack.

**Review by agniya-i — CHANGES_REQUESTED** [review]
There are still some failed tests, try to fix them 

Follow previous comments and pay attention that link should not have left/right padding, and should have the same height as the container
Screenshot 2023-03-06 at 18 23 03

File: src/style.css [hunk 6]
```diff
@@ -1 +1,65 @@
...
+.link__home {
+  padding: 10px 0 6px;
+}
+
+.nav__list {
+  display: flex;
+  justify-content: space-between;
+  flex-wrap: wrap;
+  align-content: center;
+  width: 641px;
```
Comment (polosanya) [inline src/style.css:30]: Same here. I don't think that it's a good idea to hardcore this value. Just add the correct margins between nav items

**Review by polosanya — CHANGES_REQUESTED** [review]

**Review by anstsot — CHANGES_REQUESTED** [review]
Looks good from code perspective but you still have failed tests. Try to check the font

**Review by anstsot — APPROVED** [review]
The tests seems to be failed still but I'm approving the PR

**Pull request closed without merging**